visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.ts

```diff
@@ -65,7 +65,9 @@ export class ScreenshotButtonComponent {
 		this.buildScreenShotCanvas(renderer)
 
 		renderer.domElement.toBlob(blob => {
-			navigator.clipboard.write([new ClipboardItem({ [blob.type]: blob })])
+			navigator.clipboard.write([new ClipboardItem({ [blob.type]: blob })]).catch(() => {
+				// We can add a popup on failure here, explaining the problem
```

Comment:
This will just ignore it for now and likely, no one is going to look at it from then on anymore?